src/internal/value/value.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/zarf-dev/zarf/src/pkg/logger"
 )
 
+const SchemaFileName = "values.schema.json"
```

Comment:
Should this be defined in layout such that SDK consumers have a consistent experience for how to target the schema file (should it exist)?

Reply:
I would second it being placed in the layout package. The value package shouldn't need to know what a Zarf package is

Reply:
oh that's weird, must've gotten undone in my editor. fixed